### PR TITLE
Add support for retreiving paginated data for list_comments() method

### PR DESCRIPTION
**Pull request by jatzz10**
Implements #53

### CODE REVIEW

File: freshdesk/v2/api.py [hunk 1]
```diff
@@ -181,12 +181,23 @@ class CommentAPI(object):
     def __init__(self, api):
         self._api = api
 
-    def list_comments(self, ticket_id):
-        url = "tickets/%d/conversations" % ticket_id
+    def list_comments(self, ticket_id, **kwargs):
+        url = "tickets/%d/conversations?" % ticket_id
+        page = 1 if not "page" in kwargs else kwargs["page"]
```
Comment (sjkingo) [inline freshdesk/v2/api.py:186]: Might be cleaner to go:

```
page = kwargs.get("page", 1)
```

File: freshdesk/v2/api.py [hunk 2]
```diff
@@ -181,12 +181,23 @@ class CommentAPI(object):
...
+        per_page = 100 if not "per_page" in kwargs else kwargs["per_page"]
```
Comment (sjkingo) [inline freshdesk/v2/api.py:187]: And

```
per_page = kwargs.get("per_page", 100)
```

**Comment by sjkingo** [comment]
Looks great! Just a couple of nitpicks :-)

File: freshdesk/v2/api.py [hunk 3]
```diff
@@ -181,12 +181,23 @@ class CommentAPI(object):
...
+        # Skip pagination by looping over each page and adding comments if 'page' key is not in kwargs.
+        # else return the requested page and break the loop
+        while True:
+            this_page = self._api._get(url + "page=%d&per_page=%d" % (page, per_page), kwargs)
+            comments += this_page
+            if len(this_page) < per_page or "page" in kwargs:
+                break
+            page += 1
+
+        return [Comments(**c) for c in comments]
```
Comment (sjkingo) [inline freshdesk/v2/api.py]: `s/Comments/Comment`

**Comment by sjkingo** [comment]
Thanks! Have released this as 1.3.1 on pypi.

**Comment by jatzz10** [comment]
Awesome! Thanks :)

**Pull request merged**